libs/akita/src/lib/filterNil.ts

```diff
@@ -1,10 +1,10 @@
 import { Observable } from 'rxjs';
 import { filter } from 'rxjs/operators';
-import { Diff } from './types';
 
 /**
  * @example
  *
  * query.selectEntity(2).pipe(filterNil)
  */
-export const filterNil = <T>(source: Observable<T | undefined | null>) => source.pipe(filter((value): value is Diff<T, null | undefined> => value !== null && typeof value !== 'undefined'));
```

Comment:
I think we should deprectae this version and create a function version:

```ts
export function filterNilValue<T>(): OperatorFunction<T, NonNullable<T>> {
  return filter((value: T): value is NonNullable<T> => value === null || value === undefined);
}
```

Can you please do this?

Reply:
Shouldn't it be `value !== null && value !== undefined`?

Reply:
Yes, :)